R/ecoretriever.R

```diff
@@ -136,11 +136,37 @@ datasets = function(){
   system('retriever ls', intern = TRUE) 
 }
 
+#' Update the retrievers download script to the most recent official release.
```

Comment:
"download script to the most recent official release" -> "dataset scripts to the most recent versions"